### DISCUSSION TITLE
Opening Homer in Browser results in {"message":"Not Found"}

### DISCUSSION THREAD

**Discussion by eq-spo** [sipcapture/homer-app#512] [Q&A] [answered]
Hi everyone, we are working on this problem since an automated update on the system (homer-app version 1.4.37 to 1.4.50) a few days ago but cant get it to work again. It worked great before, but downgrading to version 1.4.37 doesnt help either)

When opening Homer in a browser, we are only getting an **{"message":"Not Found"}**. When opening the Page (over 9080 or 443, doesnt matter), there is also nothing logged in the homer-app log-file, log level set to "tracer".

We tried to purge the homer-app Package and reinstalling it, using a completely fresh webapp_config.json (from the example) but the error still persists.

the heplify-server still works just fine, as grafana is logging the traffic just fine.

Does anyone have any idea what could have happened? I am very grateful for any help, no matter how small

**Comment by eq-spo** [comment, marked as answer]
We found the solution! If anyone is having the same problem:

After an update from 1.4.37 to 1.4.50 the /dist folder was quite empty, only some folders were left. We downloaded the homer-ui.xxx.tgz from the latest release and copied the content to the /dist folder and everything seems to work again.

**Reply by kYroL01** [reply]
Thank you very much for the report!
